### PR TITLE
macro-editor:add abstract; regenerate website

### PR DESCRIPTION
**Pull request by pireddag**

### CODE REVIEW

**Comment by pireddag** [comment]
I have used the update tool you announced in #35 .
I have also added `export LC_ALL=POSIX` at the beginning of `update-website`. I verified that it does not modify the environment variable in the shell where I call the script from (with `./update-website`).
The list of modification times for the articles seems ok at a quick glance.

File: update-website [hunk 1]
```diff
@@ -1,2 +1,3 @@
+export LC_ALL=POSIX
```
Comment (mgubi) [inline update-website:1]: Why is this needed? Maybe it would be useful to add a comment to explain the aim of this setting.

Comment (pireddag) [reply update-website:1]: Because (as far as I see) it is the way to set the locale for the rest of the script (which otherwise is run in the locale of the shell, which in my case isn't English).
I am adding a comment.

Comment (pireddag) [reply update-website:1]: Let me know if the comment is too long or not clear enough.

Comment (mgubi) [reply update-website:1]: You are right. I was missing this point. Thanks!

**Review by mgubi — APPROVED** [review]
